fastlane/lib/fastlane/swift_fastlane_function.rb

```diff
@@ -96,15 +96,15 @@ def determine_type_from_override(type_override: nil, default_type: nil)
       elsif type_override == Boolean
         return "Bool"
       elsif type_override == :string_callback
-        return "((String) -> Void)"
+        return "@escaping (String) -> Void"
       else
         return default_type
       end
     end
 
     def override_default_value_if_not_correct_type(param_name: nil, param_type: nil, default_value: nil)
       return "[]" if param_type == "[String]" && default_value == ""
-      return "{_ in }" if param_type == "((String) -> Void)"
+      return "{_ in }" if param_type == "@escaping (String) -> Void"
```

Comment:
I think I remember doing `((String) -> Void)` because later on, it might be made optional by adding `?` for `((String) -> Void)?`

So *_I think_* if you change this to `@escaping (String) -> Void` there might be a point where it turns into `@escaping (String) -> Void?` which isn't what you intend.